server/graphql-engine.cabal

```diff
@@ -192,6 +193,9 @@ library
                      , QuickCheck
                      , generic-arbitrary
 
+                     -- scheduled triggers
+                     , cron
```

Comment:
We should probably declare a lower bound on `cron`, just to be safe. I’d do `cron >=0.6.2`.